states/NC/governor.yaml

```diff
@@ -0,0 +1,121 @@
+          value: "$ADDRESS_STATE_POSTAL_ABBREV"
+          required: true
+          options:
+            "AL": "Alabama"
+            "AK": "Alaska"
+            "AS": "American Samoa"
+            "AZ": "Arizona"
+            "AR": "Arkansas"
+            "CA": "California"
+            "CO": "Colorado"
+            "CT": "Connecticut"
+            "DE": "Delaware"
+            "DC": "District of Columbia"
+            "FL": "Florida"
+            "GA": "Georgia"
+            "GU": "Guam"
+            "HI": "Hawaii"
+            "ID": "Idaho"
+            "IL": "Illinois"
+            "IN": "Indiana"
+            "IA": "Iowa"
+            "KS": "Kansas"
+            "KY": "Kentucky"
+            "LA": "Louisiana"
+            "ME": "Maine"
+            "MH": "Marshall Islands"
+            "MD": "Maryland"
+            "MA": "Massachusetts"
+            "MI": "Michigan"
+            "MN": "Minnesota"
+            "MS": "Mississippi"
+            "MO": "Missouri"
+            "MT": "Montana"
+            "NE": "Nebraska"
+            "NV": "Nevada"
+            "NH": "New Hampshire"
+            "NJ": "New Jersey"
+            "NM": "New Mexico"
+            "NY": "New York"
+            "NC": "North Carolina"
+            "ND": "North Dakota"
+            "MP": "Northern Marianas Islands"
+            "OH": "Ohio"
+            "OK": "Oklahoma"
+            "OR": "Oregon"
+            "PW": "Palau"
+            "PA": "Pennsylvania"
+            "PR": "Puerto Rico"
+            "RI": "Rhode Island"
+            "SC": "South Carolina"
+            "SD": "South Dakota"
+            "TN": "Tennessee"
+            "TX": "Texas"
+            "UT": "Utah"
+            "VT": "Vermont"
+            "VI": "Virgin Islands"
+            "VA": "Virginia"
+            "WA": "Washington"
+            "WV": "West Virginia"
+            "WI": "Wisconsin"
+            "WY": "Wyoming"
+    - fill_in:
+        - name: "submitted[email]"
+          selector: "#edit-submitted-email"
+          value: $EMAIL
+          required: true
+    - check:
+        - name: "submitted[does_this_email_require_a_response]"
+          selector: "#edit-submitted-does-this-email-require-a-response-1"
+          value: "1"
+          required: true
+    - fill_in:
+        - name: "submitted[phone_number]"
+          selector: "#edit-submitted-phone-number"
+          value: "$PHONE"
+    - check:
+        - name: "submitted[issue]"
+          selector: "#edit-submitted-issue-11"
+          value: "11"
+          required: true
+    - fill_in:
+        - name: "submitted[other_issue]"
+          selector: "#edit-submitted-other-issue"
+          value: "$SUBJECT"
+        - name: "submitted[message]"
+          selector: "#edit-submitted-message"
+          value: "$MESSAGE"
+          require: true
+    - recaptcha:
+        - value: true
+    - click_on:
+        - value: "Submit"
+          selector: "#webform-client-form-11 > div.form-actions > input"
```

Comment:
Hehe brilliant that the form has two `input[value=Submit]`. This is fine, and I wonder if this might be more robust to the form structure changing in the future.
```
"input[value=Submit][name=op]"
```

Reply:
Yeah that would be more resistant to change.  I'll log an issue and put it in a PR for one of the other new states.